### PR TITLE
[Call][Release] calling-v1.2.0-beta.1

### PR DESCRIPTION
**Pull request by iaulakh**

### CODE REVIEW

**Review by Mohtasim — DISMISSED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Review by ShaunaSong — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ android {
 ```groovy
 dependencies {
     ...
-    implementation 'com.azure.android:azure-communication-ui-calling:1.1.0'
+    implementation 'com.azure.android:azure-communication-ui-calling:<latest stable release version>'
```
Comment (pavelprystinka) [inline README.md:37]: We need to revert this change. It should state 1.1.0 and do not point to releases GitHub page

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iaulakh) [reply README.md]: done

**Review by pavelprystinka — APPROVED** [review]

**Review by ShaunaSong — APPROVED** [review]
